tests/varispeed_test.c

```diff
@@ -144,8 +152,101 @@ varispeed_test (int converter, double target_snr)
 		exit (1) ;
 		} ;
 
-	puts ("ok") ;
-
 	return ;
 } /* varispeed_test */
 
+static void
+varispeed_hammer_test (int converter)
+{	double ratios [] = { 1.0, 0.01, 20 } ;
```

Comment:
Missing corner case ratios. Adding `1./256.` shows a heap-user-after-free in ASAN. Also the initial test case as reported in #5 for ratios `0.09 -> 0.08` is missing (or more generally 2 ratios below 1 as e.g. `0.1 and 0.01` which was contained in my test)